enos/enos-variables.hcl

```diff
@@ -200,3 +200,33 @@ variable "hcp_boundary_cluster_id" {
   // If using HCP int, ensure that the cluster id starts with "int-"
   // Example: "int-19283a-123123-..."
 }
+
+variable "gcp_target_instance_type" {
+  description = "Instance type for test target nodes"
+  type        = string
+  default     = "e2-micro"
+}
+
+variable "gcp_region" {
+  description = "GCP region where the resources will be created"
+  type        = string
+  default     = "us-central1"
+}
+
+variable "gcp_zone" {
+  description = "GCP zone where the resources will be created"
+  type        = string
+  default     = "us-central1-a"
+}
+
+variable "gcp_project_id" {
+  description = "GCP project where the resources will be created"
+  type        = string
+  default     = ""
+}
+
+variable "gcp_client_email" {
+  description = "GCP client email"
+  type        = string
+  default     = ""
+}
```

Comment:
Is this needed? It seems like it gets automatically set in `gcp_iam_setup`?

Reply:
This is required to set the client email variable here: https://github.com/hashicorp/boundary/blob/6ed1fab838e440242b169b626486ed3f1c86de37/enos/enos-scenario-e2e-docker-base-with-gcp.hcl#L151

Unless the enos tests do not work that way?

Reply:
If I'm understanding right...

- the enos-variables.hcl defines some "global" variables that are can be populated by a user prior to kicking off enos (for example, by doing `export ENOS_VAR_my_variable=123`). This would allow you to use `var.my_variable` in a scenario file.
- For that line you referenced, you're populating the `gcp_client_email` input of one module with the output from the `gcp_iam_setup` module